### PR TITLE
Feature/sc 10147/import utilities add itemrelations productvariations

### PR DESCRIPTION
**Pull request by vsimak**
- Added SKUs to Item Relations
- Refactored items function

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -6919,6 +6919,15 @@ packages:
     resolution: {integrity: sha512-1MzNQdAvO+54H+EaK5YpyEy0T+Ejo/7YLHS93G3RnYWh5gaotGHwGeN/ZO687qEDU2y4CdStQYXVHIgrUl5UVQ==}
     dev: false
 
+<<<<<<< HEAD
```
Comment (Plopix) [inline pnpm-lock.yaml:6922]: that's probably wrong

File: src/bootstrap-tenant/json-spec/core.ts [hunk 2]
```diff
@@ -169,15 +169,23 @@ export interface JSONPropertiesTableSection {
...
   cataloguePath?: string
   externalReference?: string
+  sku?: string
 }
 export interface JSONGridReference {
   name?: string
 }
-export type JSONItemRelation = JSONItemReference
-export type JSONItemRelations = JSONItemRelation[]
+// export type JSONItemRelation = {
```
Comment (Plopix) [inline src/bootstrap-tenant/json-spec/core.ts:180]: remove?

**Review by Plopix — CHANGES_REQUESTED** [review]
awesome job here. I did not test it, but did look at the code and found some stuff to change.

**Comment by Plopix** [comment]
also how did you prepare:
```    "@crystallize/js-api-client": "^1.12.2",
    "@crystallize/schema": "^0.1.4",```
    they did not exist :) 
    
lock should be recreated anyway :)

File: pnpm-lock.yaml (hunk 1 above)
Comment (vsimak) [reply pnpm-lock.yaml]: Fixed!

**Pull request merged**